contributing-guides/style-guide.md

```diff
@@ -330,7 +330,16 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
-## Emphasis
+## Not rendered syntax
+
+Any syntax not documented in this guide or the client specification will **not** be rendered in most of our clients. See common cases below.
```

Comment:
This is unclear. What do you mean "any syntax"? Technically the client spec requires CommonMark support, even if we don't use bold/italics etc by convention.....

Reply:
> This is unclear. What do you mean "any syntax"? Technically the client spec requires CommonMark support, even if we don't use bold/italics etc by convention.....

Maybe, we could reword this description to include CommonMark too.